rasa/cli/evaluate.py

```diff
@@ -80,7 +80,7 @@ def add_subparser(
     marker_parser.set_defaults(func=_run_markers_cli)
 
 
-def _run_markers_cli(args: argparse.Namespace) -> None:
+async def _run_markers_cli(args: argparse.Namespace) -> None:
```

Comment:
the CLI needs a non `async` entrypoint, otherwise things will not be executed

Reply:
Got it! Addressed